tracing-subscriber/src/subscribe/mod.rs

```diff
@@ -1690,7 +1693,8 @@ feature! {
         }
 
         fn max_level_hint(&self) -> Option<LevelFilter> {
-            let mut max_level = LevelFilter::ERROR;
+            // Default to `OFF` if there are no underlying `Subscribe`s
```

Comment:
```suggestion
            // Default to `OFF` if there are no underlying subscribers.
```
normally we just pluralize this to the actual plural form of the word, ("subscribes" makes me mildly uncomfortable...)